test/e2e/aadpodidentity_test.go

```diff
@@ -386,7 +386,7 @@ var _ = Describe("Kubernetes cluster using aad-pod-identity", func() {
 
 		// Delete pod identity to verify that the VM identity did not get deleted
 		waitForDeployDeletion(identityValidator)
-		cmd := exec.Command("kubectl", "delete", "AzureIdentity,AzureIdentityBinding,AzureAssignedIdentity", "--all")
```

Comment:
Is it possible to add this as separate commit so that we can track back why we changed this ?

Reply:
Done.